test/testenv/verificationutils.go

```diff
@@ -59,6 +59,25 @@ type PodDetailsStruct struct {
 	} `json:"status"`
 }
 
+// VerifyMCReady verify Standlone is in ReadyStatus and does not flip-flop
```

Comment:
VerifyMCReady verify "MonitoringConsole" is in ReadyStatus and does not flip-flop